acceptance/tests/pxp-module-puppet/negative/attempt_puppet_flag_whitelist_bypass.rb

```diff
@@ -0,0 +1,58 @@
+require 'pxp-agent/test_helper.rb'
+
+test_name 'C100245 - attempt bypass of flag whitelist with tab character' do
+
+  confine :except, :platform => ['windows']
+
+  step 'pick test host' do
+    @test_host = agents.first
+  end
+
+  step 'set up executable file' do
+    @executable_path = [@test_host.system_temp_path, 'run_me'].join('/')
```

Comment:
I'd suggest using [`File::join`](https://ruby-doc.org/core-2.4.0/File.html#method-c-join) instead of `Array#join`.